homeassistant/components/overkiz/water_heater_entities/hitachi_dhw.py

```diff
@@ -0,0 +1,101 @@
+"""Support for Hitachi DHW."""
+from __future__ import annotations
+
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.water_heater import (
+    STATE_HIGH_DEMAND,
+    WaterHeaterEntity,
+    WaterHeaterEntityFeature,
+)
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    PRECISION_WHOLE,
+    STATE_OFF,
+    STATE_ON,
+    TEMP_CELSIUS,
+)
+
+from ..entity import OverkizEntity
+
+OVERKIZ_TO_OPERATION_MODE: dict[str, str] = {
+    OverkizCommandParam.STANDARD: STATE_ON,
+    OverkizCommandParam.HIGH_DEMAND: STATE_HIGH_DEMAND,
+    OverkizCommandParam.STOP: STATE_OFF,
+}
+
+OPERATION_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_OPERATION_MODE.items()}
+
+
+class HitachiDHW(OverkizEntity, WaterHeaterEntity):
+    """Representation of Hitachi DHW."""
+
+    _attr_min_temp = 30.0
+    _attr_max_temp = 70.0
```

Comment:
Just to confirm: there is no way of getting those programmatically ?

Reply:
@Quentame Yes, no way to get it programmatically for this device.
In the only overkiz setup I found using this device, there is no indication of these values.  
These values was set like that in the ha-tahoma code.